### ISSUE TITLE
A question about the Comformer test results

### ISSUE THREAD

**Issue by JCui-xmu** [divelab/AIRS#31]
Hi, Comformer is a great work, but I have a question about the test results, in fact, matformer also experienced such problems.
When I run train_jarvis/mp.py to test the trained model, a difference in one parameter will lead to different results, as shown below

`train_prop_model(learning_rate=0.0005, name="iComformer", scheduler="polynomial", prop=props[10], pyg_input=True, n_epochs=800, max_neighbors=25, cutoff=4.0, batch_size=64, use_lattice=True, output_dir="./jarvis_iComformer_Total_energy", use_angle=False, save_dataloader=False/True, test_only=True)`

When I clear the cached data generated during training and set save_dataloader to False, I get similar results to the paper. However, when this parameter is set to True and the cached data is used in subsequent tests, the results are very different. Here is an example

I tested the total energy of iComformer on Jarvis and got a result of 0.013602, which is much different from the 0.0288 in the paper. Other properties also have similar problems. May I ask if I did something wrong?

0.013602 vs 0.0288

**Comment by YKQ98** [comment]
Yes, please set save_dataloader to False whenever possible. This setting is for the usage of tuning parameters. All data should be re-generated for different tasks.
